src/widget/about/aboutfriendform.cpp

```diff
@@ -2,6 +2,7 @@
 #include "src/widget/gui.h"
 #include "ui_aboutfriendform.h"
 #include "src/core/core.h"
+#include "../style.h"
```

Comment:
Please, use full path, if file is not in the same directory. I.e. `#include "src/widget/style.h"`